superset-frontend/cypress-base/cypress/e2e/dashboard/drilltodetail.test.ts

```diff
@@ -249,9 +249,13 @@ describe('Drill to detail modal', () => {
       it('opens the modal with the correct filters', () => {
         interceptSamples();
 
+        // focus on table first to trigger browser scroll
+        cy.get("[data-test-viz-type='table']").contains('boy').rightclick();
+
+        cy.wait(500);
```

Comment:
I wonder if there's something we can do with timeouts instead of waiting, but... probably not worth the hassle.

Reply:
Yeah this was really tricky because the browser scrolls automatically on focus, and then the field goes under the header and the right click doesn't work. I adjusted for this behavior for real users in the onFocus below but for automations, the wait was the only way to get the browser to catch up. But, that said, there are a few waits earlier in this script that I think we can change to timeouts/assert.is.visible, etc.

Reply:
let's see if the above changes make this a net neutral/positive.